test.py

```diff
@@ -406,15 +406,15 @@ async def create_cluster(logger: Union[logging.Logger, logging.LoggerAdapter]) -
             cluster = ScyllaCluster(logger, self.hosts, cluster_size, create_server)
 
             async def stop() -> None:
-                await cluster.stop()
+                await cluster.stop_gracefully()
```

Comment:
If we want this to be the default, we should make `stop()` and `uninstall()` be graceful and add a `stop_abruptly()` and `uninlstall_abruptly()` for when/if needed. Otherwise new test suites might default to the undesired behaviour.

Reply:
You are right, I will change this.

Reply:
@denesb on a second thought making this change is going to be quite extensive and touch many files and lines of code so I think I will postpone it to a followup patch.
```
 grep -rIl --include="*.py" --exclude-dir=".devcontainer" stop_gracefully | wc -l
24
```
This is just an estimation since stop (not gracefully) also exists, but we have stop on other objects to so the search is more involved.

Reply:
Ok, I'm fine with a follow-up.